content/help/system-requirements.adoc

```diff
@@ -153,6 +153,9 @@ the following window managers:
 * i3 (Arch Linux)
 * Unity (Ubuntu prior to 18.04)
 
+MATE is not officially supported, but works well in most cases. Check for some issues  
```

Comment:
I'm not comfortable listing things that may work here.  This is for requirements.  Lots of distributions will work but we cannot support them without a developer willing to take on the bug reports.

Reply:
I have reworded it now so that it is clear that it is not to be expected to work. How do you like it now?

Reply:
No, it should not be mentioned at all here.  The requirements are only for supported systems.